### ISSUE TITLE
Feature Request: slot the event to modify it's html

### ISSUE THREAD

**Issue by Sunhat** [CroudTech/vue-fullcalendar#125]
Hi,

just a feature request :) It'd be great to do something like

```
<full-calendar>
    <template slot-scope="event">{{ event.name }}</template>
</full-calendar>
```

**Comment by Sunhat** [comment]
Perhaps in the future? I saw you're moving to an alpha version without jQuery? 

Not the most pressing issue. Although it would make implementing things like qtip a lot easier

**Comment by BrockReece** [comment]
Yeah possibly, I guess it depends how it handles its Dom as to whether it would be compatible.

**Comment by BrockReece** [comment]
I just had a quick play around with this and managed to build the event element from a Vue component. in the eventRender method.

```js
// App.vue
import CalendarEvent from './components/CalendarEvent'
...
config: {
  eventRender(event, element) {
    const Constructor = Vue.extend(CalendarEvent);
    const vm = new Constructor({
      propsData: {
        event
      }
    }).$mount();

    element.html(vm.$el);
  },
}
```
What do you think?
Here is the full [sandbox](https://codesandbox.io/s/32kjlpj9z1)

**Comment by plavet** [comment]
How could I emit click action from CalendarEvent to App.vue, from child to parent, in this case? I have a method on CalendarEvent with this.$emit('edit-current-event'), but where do I write @edit-current-event="someAction" on parent? App.vue in this case, I've forked your sandbox example and made some changes to ilustrate what i need to do, [sandbox](https://codesandbox.io/s/rwk45o3w2p)

**Comment by ncbinh98** [comment]
> I just had a quick play around with this and managed to build the event element from a Vue component. in the eventRender method.
> 
> ```js
> // App.vue
> import CalendarEvent from './components/CalendarEvent'
> ...
> config: {
>   eventRender(event, element) {
>     const Constructor = Vue.extend(CalendarEvent);
>     const vm = new Constructor({
>       propsData: {
>         event
>       }
>     }).$mount();
> 
>     element.html(vm.$el);
>   },
> }
> ```
> 
> What do you think?
> Here is the full [sandbox](https://codesandbox.io/s/32kjlpj9z1)

This works fine **but** the **performance** is the issue, every time I manipulate it will increase the buffer a lot, after 5 or 6 times CRUD events, my memory is up to 80% even 90% if I continue.
